components/openzwave-adapter/src/lib.rs

```diff
@@ -490,11 +490,11 @@ impl taxonomy::adapter::Adapter for OpenzwaveAdapter {
         }).collect()
     }
 
-    fn register_watch(&self, mut values: Vec<(TaxoId<Channel>, Option<Value>, Box<ExtSender<WatchEvent>>)>) -> Vec<(TaxoId<Channel>, Result<Box<AdapterWatchGuard>, TaxoError>)> {
+    fn register_watch(&self, mut values: Vec<(TaxoId<Channel>, Option<Value>, Box<ExtSender<WatchEvent<Value>>>)>) -> Vec<(TaxoId<Channel>, Result<Box<AdapterWatchGuard>, TaxoError>)> {
```

Comment:
Because we use `WatchEvent<Value>` everywhere, maybe it would make sense to either not change this now (maybe you'll need it later, then change it later ? or even in a separate patch if you really want it), or add a type alias.